src/game/client/hud/jumpspeed.cpp

```diff
@@ -47,10 +49,19 @@ void CHudJumpspeed::Draw(float flTime)
 		a = MIN_ALPHA;
 
 	int y;
-	if (hud_jumpspeed_below_cross.GetBool())
-		y = ScreenHeight / 2 + gHUD.m_iFontHeight / 2 + gHUD.m_iFontHeight;
-	else
+	if (!hud_jumpspeed_below_cross.GetBool())
+	{
 		y = ScreenHeight - gHUD.m_iFontHeight - gHUD.m_iFontHeight / 2;
+	}
+	else
+	{
+		int yoffset = 0;
+
+		if (hud_jumpspeed_below_cross.GetBool() && hud_speedometer_below_cross.GetBool() && hud_speedometer.GetBool())
```

Comment:
`hud_jumpspeed_below_cross.GetBool()` is always true in the else branch so it's not needed.

Reply:
`hud_speedometer_below_cross.GetBool()` and `hud_speedometer.GetBool()` should be rearranged so they are consistent with speedometer code
```cpp
// speedometer.cpp
		if (hud_jumpspeed.GetBool() && !hud_jumpspeed_below_cross.GetBool())
			yoffset = gHUD.m_iFontHeight;
```